public/styles/timeline.css

```diff
@@ -1,160 +1,492 @@
+    float: right;
+  }
+  summary:hover {
+    background-color: lightgrey;
+  }
+  @media (hover: none) {
+    summary:hover {
+      background-color: rgb(255 190 0);
     }
+  }
+
+  #timeline__typefilter {
+    border: 0.5rem solid white;
+    border-radius: 1rem;
+    padding: 1rem;
+    width: 100%;
+
+    legend {
+      background-color: #11ddee;
+      border: 5px solid #00144b;
+      box-shadow: 2px 1px black;
+      padding-inline: 1rem;
 
-    .entry-description {
-      padding: 1rem;
+      label {
+        width: 100%;
+        display: inline-block;
+        font-size: 1.5rem;
+        font-weight: 700;
+        line-height: 2;
+      }
     }
   }
 }
 
-.timeline-nav {
-  position: sticky;
-  bottom: 0;
-  display: flex;
-  gap: 2rem;
-  overflow-x: auto;
-  background-color: #f02017;
-  border-top: 2pt solid black;
-
-  .timeline-year {
+.event_type_pill {
+  display: inline-flex;
+  --pill-font-size: 1.5rem;
+  background-color: #fb46ce;
+  cursor: pointer;
+  border: 5px solid #00144b;
+  box-shadow: 2px 1px black;
+  margin: 0.5rem;
+  font-size: var(--pill-font-size);
+  font-weight: 600;
+  padding-inline: 0.5rem;
+
+  color: black;
+  text-align: center;
+  width: fit-content;
+
+  border: 5px solid #00144b;
+  box-shadow: 2px 1px black;
+
+  input[type="checkbox"] {
+    appearance: none;
+    background-color: inherit;
+    margin: 0;
+    font: inherit;
+    color: currentColor;
+    width: var(--pill-font-size);
+    height: var(--pill-font-size);
+    border: 0.15em solid currentColor;
+    border-radius: 0.15em;
+    margin-top: 0.3rem;
     display: grid;
-    grid-template-rows: 1fr 1fr;
-    gap: 0.5rem;
-    padding: 0.5rem 1rem;
-    border-left: 2pt solid black;
+    place-content: center;
+  }
+  input[type="checkbox"]::before {
+    content: "";
+    width: 0.65rem;
+    height: 0.65rem;
+    transform: scale(0);
+    transition: 120ms transform ease-in-out;
+    box-shadow: inset 1rem 1rem black;
+  }
+  input[type="checkbox"]:checked::before {
+    transform: scale(1);
+  }
 
-    &:first-child {
-      border-left: none;
-    }
+  span {
+    display: inline-block;
+    width: fit-content;
+    text-align: left;
+    margin-inline: 0.5rem;
+  }
+}
+.event_type_pill:hover {
+  background-color: lightgrey;
+}
+@media (hover: none) {
+  .event_type_pill:hover {
+    background-color: #fb46ce;
+  }
+}
 
-    >span {
-      color: yellow;
-      font-size: 2rem;
-      font-weight: 800;
-      font-family: "Haros Baelz";
-      text-shadow: 2px 2px black;
-      text-decoration: none;
-    }
+#timeline {
+  grid-area: timeline;
+  font-family: "Noto Sans JP", "Noto Sans KR", sans-serif;
+}
+
+.timeline__event {
+  display: grid;
+  grid-auto-flow: dense;
+  grid-template-columns: minmax(1rem, 1fr) minmax(20rem, 3fr) 0.5rem minmax(20rem, 3fr) minmax(1rem, 1fr);
+  grid-template-areas: "... left line right ...";
+  column-gap: 2rem;
+
+  .aussie {
+    transform: scale(-1);
+    text-align: left !important;
```

Comment:
You could increase the specificity of this by doing `.timeline__info.aussie`

Reply:
Not at the moment because `.aussie` gets applied to both `timeline__info` and `timeline__media` since the Tweet screenshot getting flipped so the "wassup" is readable normally is part of the joke. We could flip the image we host so the class is only required on `.timeline__info`.